ios/PurchasesHybridCommon/ObjCAPITester/RCHybridCommonAPITest.m

```diff
@@ -151,6 +151,8 @@ - (void)testSubscriberAttributes {
     [RCCommonFunctionality setMparticleID:nil];
     [RCCommonFunctionality setOnesignalID:@""];
     [RCCommonFunctionality setOnesignalID:nil];
+    [RCCommonFunctionality setOnesignalUserID:@""];
+    [RCCommonFunctionality setOnesignalUserID:nil];
```

Comment:
Why this pattern?

Reply:
These API tests basically make sure that the APIs don't change inadvertently. We only perform some basic verification (compile them) right now. So these are basically making sure that there is a `setOnesignalUserID` method that accepts a string or nil.

This new method is needed when using OneSignal SDK 11+, when using versions <=9, developers should still use the existing methods.